raster-foundry/raster-foundry
# Add a static page

## Overview

This PR adds a static page for github pages to show app is not available and provide a button to the marketing site.

### Checklist

- ~Swagger specification updated~
- ~New tables and queries have appropriate indices added~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~
- ~Any new endpoints have scope validation and are included in the integration test csv~

### Demo

![Screen Shot 2021-07-26 at 5 01 29 PM](https://user-images.githubusercontent.com/16109558/127058611-d9e0c9f0-7080-4dec-8678-7fa2c5d6c280.png)


## Testing Instructions

- If the `test` branch is not overwritten, https://app.staging.rasterfoundry.com/ should show a spinning wind turbine, new copy, and a button to the RF marketing site
- If it is overwritten, push up a test branch and make sure you see the above mentioned page

Closes https://github.com/azavea/raster-foundry-platform/issues/1307
